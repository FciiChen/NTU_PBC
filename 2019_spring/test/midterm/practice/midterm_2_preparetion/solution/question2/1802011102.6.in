jYcvkh9C, j3adjHd,q3fhw0b,f5yitXJ6,Ewq,2dIOnwqg
o1agh7, AKzmxzai,rgXHcJHTgB,Zy6u1i5Z,uwRPXdRQY,GwTaxMB9
BREAK
o1agh7 ,  bXWeCH
jYcvkh9C ,  c73OrbF
BREAK
