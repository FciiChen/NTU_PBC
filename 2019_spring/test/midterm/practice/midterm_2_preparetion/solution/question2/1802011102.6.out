jYcvkh9C   j3adjHd    q3fhw0b    f5yitXJ6   Ewq        2dIOnwqg   c73OrbF   
o1agh7     AKzmxzai   rgXHcJHTgB Zy6u1i5Z   uwRPXdRQY  GwTaxMB9   bXWeCH    
